core/src/main/java/net/opentsdb/query/processor/groupby/GroupBy.java

```diff
@@ -68,6 +65,8 @@
    */
   private DownsampleConfig downsampleConfig;
 
+  private int timeSeriesCount;
```

Comment:
Unfortunately we can't store this here as a GB may get multiple results (depend on how the user configures their graph). Instead we need to pass the Result into the GB result as a source result, then let the GB Iterators read from that.